docs/faq.md

```diff
@@ -14,11 +14,11 @@ Currently this is a manual process. We are working on dashboards and other metri
 
 **Does MUO reserve compute capacity?**
 
-Yes, if `capacityReservation: true` is enabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.
+Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO will create a temporary +1 to every worker `machinesets` within the cluster. In multi availability zones, this is true for each zone.
 
 **Does MUO maintain correct instance types for each machine pool?**
 
-Yes, if `capacityReservation: true` is enabled in the upgradeconfig CR. MUO creates the extra compute based on the found instance types of the current `machinesets`.
+Yes, if `capacityReservation` is not disabled in the upgradeconfig CR. MUO creates the extra compute based on the found instance types of the current `machinesets`.
```

Comment:
```suggestion
Yes, if `capacityReservation` in the upgradeconfig CR is set to `true`. MUO creates the extra compute based on the found instance types of the current `machinesets`.
```

Reply:
Thanks @T0MASD for review. I have updated the doc based on your suggestion.